automatic/clementine/tools/chocolateyInstall.ps1

```diff
@@ -1,30 +1,27 @@
-$ErrorActionPreference = 'Stop'
+﻿$ErrorActionPreference = 'Stop';
```

Comment:
Why adding a semicolone? Since there isn't a semicolone used anywhere else IMHO one shouldn't be used here either